CONTRIBUTORS.rst

```diff
@@ -27,12 +27,14 @@ Listed in alphabetical order.
 ================ ===============
 Name             GitHub
 ================ ===============
+Alan Moffet `@amoffet`_
```

Comment:
For the table to render, GitHub username needs to align with those below (i.e., add spaces before).